EIPS/eip-zodiac.md

```diff
@@ -0,0 +1,175 @@
+---
+eip: <to be assigned>
+title: Zodiac
+description: A composable design philosophy for DAOs.
+author: Auryn Macmillan (@auryn-macmillan), Kei Kreutler (@keikreutler)
+discussions-to: https://gnosisguild.mirror.xyz/OuhG5s2X5uSVBx1EK4tKPhnUc91Wh9YM0fwSnC8UNcg
+status: Draft
+type: Standards Track
+category (*only required for Standards Track): ERC
+created: 2022-04-14
```

Comment:
`requires` should be added, such as "requires: 165"

Reply:
Added.